### PR TITLE
[cs] Implement cs.Syntax.code()

### PR DESCRIPTION
**Pull request by kLabz**
_(and also `cs.Syntax.plainCode()`)_

### CODE REVIEW

File: std/cs/Syntax.hx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+@:noClosure
+extern class Syntax {
+	/**
+		Inject `code` directly into generated source.
+
+		`code` must be a string constant.
+
+		Additional `args` are supported to provide code interpolation, for example:
+		```haxe
+		Syntax.code("System.Console.WriteLine({0}, {1})", "hi", 42);
```
Comment (kLabz) [inline std/cs/Syntax.hx]: Ok this example is bad for c# x) This doesn't error, but `42` is just ignored at runtime

File: std/cs/Syntax.hx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ */
+
+package cs;
+
+import haxe.extern.Rest;
```
Comment (Gama11) [inline std/cs/Syntax.hx]: `haxe.extern.Rest` is more or less deprecated in favor of `haxe.Rest` from what I gathered, so it probably shouldn't be used for new code.

**Comment by kLabz** [comment]
Just rebased development but I don't have much hope for CI atm =/

Could this make it to 4.2? (with more work if needed) This would allow me to cleanup things where I currently can't do some things with haxe syntax.

**Comment by RealyUniqueName** [comment]
I'd merge this for 4.2 once CI settles down.

**Comment by kLabz** [comment]
@RealyUniqueName sorry to bother you again, but I'm preparing our CI and docker images for 4.2 and I just realized this PR was not merged so our projects don't compile with my images using latest development =/ Could this be merged now that CI is green please?

**Pull request merged**